gcloud/bigtable/happybase/test_batch.py

```diff
@@ -352,7 +486,7 @@ def test_it(self):
         expected_result = [
             ['cf1', None],
             ['cf2', None],
-            ['cf3', ''],
+            ['cf3', ''],  # AGGGHHHHHHHHHH THIS IS WEIRD
```

Comment:
@jonparrott I forgot to ask you about this when #1533 went in. Sorry.

In my old implementation I was using `'cf3::'.rstrip(':')` but I'm thinking this should fail?